custom_components/solarman/inverter_definitions/kstar_hybrid.yaml

```diff
@@ -340,6 +340,29 @@ parameters:
         scale: 0.1
         rule: 1
         registers: [3066]
+
```

Comment:
Please, add the icon back to the main battery

Reply:
Why is that? HA assigns the icon automatically for battery class? Or am I missing something?

Reply:
> Mmm, I don't know if HA assigns an icon by default.

Yes it does, every device class in [this](https://developers.home-assistant.io/docs/core/entity/sensor/#available-device-classes) list has default icon.

Reply:
Ah, didn't know. Perfect then.